database/seed_data.js

```diff
@@ -0,0 +1,64 @@
+const Career = require('./career_model.js');
+
+const careers = [
+  new Career({ title: 'Doctor',
+    tagline: 'Promots and Improves Health',
+    description: 'A doctor is a professional who practises medicine, which is concerned with promoting, maintaining, or restoring health.',
+    image: 'https://cdn4.iconfinder.com/data/icons/avatar-male/920/Accountant_Character-512.png',
+    degree: 'Medicine',
+    grade_bagrut: '11.10',
```

Comment:
The grades are `Number` in the Schema but `String` in the database content :confused:

Reply:
Its in quotes here so its a string. However I can change them both to number?

Reply:
Yeah you're right I will change it!